src/main/java/bio/terra/app/logging/LoggerInterceptor.java

```diff
@@ -39,6 +46,20 @@ public void afterCompletion(
 
     // skip logging the status endpoint
     if (!url.endsWith("/status")) {
+
+      Map<String, Object> stackDriverPayload = new HashMap<>();
+      if ("post".equalsIgnoreCase(method)) {
+        String postBody =
+            new String(
+                    ((ContentCachingRequestWrapper) request).getContentAsByteArray(),
+                    StandardCharsets.UTF_8)
+                .lines()
+                .collect(Collectors.joining(""));
+        JsonNode postBodyJson = objectMapper.readTree(postBody);
```

Comment:
It feels dangerous and potentially heavyweight to have to parse the request twice...especially if we're truncating.  Can we just read the contents and not convert to JSON?

Reply:
I think there's an argument for converting to JSON. It allows us to drill into the post body and match on elements using stackdriver queries.

Reply:
But we talked about the fact that with truncation it won't work

Reply:
I played around with the stackdriver querying, and think that just logging the string representation of the JSON is good enough for almost all cases.